pom.xml

```diff
@@ -927,7 +928,8 @@
             <runOrder>alphabetical</runOrder>
             <systemPropertyVariables>
               <alluxio.test.mode>true</alluxio.test.mode>
-              <jacoco-agent.destfile>${project.basedir}/target/jacoco.exec</jacoco-agent.destfile>
+              <jacoco-agent.append>true</jacoco-agent.append>
+              <jacoco-agent.destfile>${build.path}/../target/jacoco-combined.exec</jacoco-agent.destfile>
```

Comment:
what does `${build.path}` normally resolve to? and how is it different from `${project.basedir}`?

Reply:
Every sub-module/pom file already defines its own `build.path`, to point to a folder in the project root directory. I am reusing that to get to a different folder (`target`) in the project root directory. `project.basedir` is relative to the specific sub-module, so each sub-module ends up writing its own file, instead of appending to the shared one.

Reply:
I see. I hope that there is a way to verify that the `build.path` is correct for each module. I'm wondering if any are incorrect by chance. If they are it could impact the coverage report

Reply:
I think if the `build.path` is incorrect, the actual compilation might fail, since that path was already being used. However, even after I ran it, I only found 1 instance of jacoco-combined.exec file, so in the current state, the `build.path` settings are all correct.